### DISCUSSION TITLE
认证代理程序无法正常工作

### DISCUSSION THREAD

**Discussion by YoungZSh** [natfrp/wiki#475] [使用问题]
![image](https://github.com/natfrp/wiki/assets/105100573/cdae0189-278a-486c-ae42-a0175bccebf2)

**Comment by fakeboboliu** [comment]
出现此错误通常表明您已经通过访问认证，无需再次执行。

请查看日志，应当可见到 `[x.x.x.x] 已被放行访问`。

为了隐私和安全考虑，相关逻辑完全在您的本地处理，并仅在您的实际请求前执行，服务端仅转发加密数据，
因此相关状态无法通知到访客程序显示为通过，给您造成的不便还请谅解。
